flow-typed/msw.js

```diff
@@ -0,0 +1,33 @@
+// @flow
```

Comment:
This probably needs a lot of improvements, but it is a start

Reply:
Can you please add there a to-do comment to move it to `flow-typed` repository where it belongs?